ansible/group_vars/go-waku.yml

```diff
@@ -40,6 +41,13 @@ waku_peers_consul_services:
   - { name: 'nim-waku-v2', env: 'wakuv2',  stage: '{{ stage }}' }
   - { name: 'go-waku',     env: 'go-waku', stage: '{{ stage }}' }
 
+# DiscV5
+go_waku_disc_v5_enabled: true
+
+# DNSDiscovery
+go_waku_dns_disc_enabled: true
+go_waku_dns_disc_url: 'enrtree://AOFTICU2XWDULNLZGRMQS4RIZPAZEHYMV4FYHAPW563HNRAOERP7C@test.waku.nodes.status.im'
```

Comment:
This is not correct (yet). We need the ENRs from go-waku to create a new tree deployed to a new domain. This will connect the go-waku fleet to the wakuv2.test fleet which we don't want afaik (it should be part of the status fleets, for which no URL exist yet).